test/helpers/restricted-nori.ts

```diff
@@ -0,0 +1,140 @@
+import type { BigNumber } from 'ethers';
+
+import type { RestrictedNORI } from '@/typechain-types/contracts/RestrictedNORI';
+import type { setupTest } from '@/test/helpers';
+import { expect, createRemovalTokenId, NOW } from '@/test/helpers';
+
+export const SECONDS_IN_1_YEAR_AVG = 31_556_952;
+export const SECONDS_IN_10_YEARS = 315_569_520;
+export const SECONDS_IN_5_YEARS = SECONDS_IN_10_YEARS / 2;
+
+export const mintAndListRemovals = async ({
```

Comment:
is this duplicating `batchMintAndListRemovalsForSale`

Reply:
haha nice, I think `batchMintAndListRemovalsForSale` was born while this PR was hanging open so we ended up coming up with the same thing ultimately.  will refactor